src/Sulu/Bundle/AdminBundle/Resources/js/components/SingleItemSelection/SingleItemSelection.js

```diff
@@ -9,6 +9,7 @@ import Button from './Button';
 import type {Button as ButtonConfig} from './types';
 
 type Props = {|
+    allowRemoveWhileDisabled: boolean,
```

Comment:
I think here this naming is a little bit confusing, therefore I would change it in all places.

At least the way I would imagine it, is that if the field itself is disabled, I cannot remove anything. But if only the item assigned to it is disabled, I would think I can still remove it.

Therfore I would name this option something like `allowRemoveForDisabledItems`.

Reply:
This is not the field, but the container. The container itself does not know if it is just the item that is disabled right now